_includes/snippets_library/TNTech_slurm/_config_options.yml

```diff
@@ -0,0 +1,68 @@
+#------------------------------------------------------------
+# Tennessee Tech University: Impulse HPC + SLURM
+#------------------------------------------------------------
+
+# Cluster host and scheduler options: the defaults come from
+# Graham at Compute Canada, running SLURM. Other options can
+# be found in the library of snippets,
+# `_includes/snippets_library`. To use one, replace options
+# below with those in `_config_options.yml` from the
+# library. E.g, to customise for Cirrus at EPCC, running
+# PBS, we could replace the options below with those from
+# 
+# _includes/snippets_library/EPCC_Cirrus_pbs/_config_options.yml
+# 
+# If your cluster is not represented in the library, please
+# copy an existing folder, rename it, and customize for your
+# installation. Remember to keep the leading slash on the
+# `snippets` variable below!
+
+snippets: "/snippets_library/TNTech_slurm"
```

Comment:
Recommended path name would be "TNTech_Impulse_slurm", to leave room for clusters at TN Tech named anything else.